internal/server/placepage/place_page.go

```diff
@@ -275,8 +279,8 @@ func fetchBtData(
 	for place, data := range mergedPlacePageData {
 		finalData := &pb.StatVarSeries{Data: map[string]*pb.Series{}}
 		categoryData[place] = &pb.Categories{}
-		categoryData[place].Category = append(categoryData[place].Category,
-			data.Categories...)
+		categoryData[place].Category = data.Categories
+		sort.Strings(categoryData[place].Category)
```

Comment:
Oh, so MergeDedupe() already sorts the result. So can just do 

```
cateogryData[place] = &pb.Categories{Category: data.Categories}
```

Reply:
The case where I saw an issue was where only one import group had categories for a place. In that case, because the code never passes through merge dedupe, it would cause output to be unsorted. I can move the sort to the if block to counter this though, if you think that would be better.

Reply:
Maybe can remove the "else" condition in line 262, so it always do MergeDedup

Reply:
Makes sense, done.